mod-circulation/src/main/resources/domain/mod-circulation/features/loans.feature

```diff
@@ -251,4 +251,35 @@ Feature: Loans tests
     * def checkInResponse = call read('classpath:domain/mod-circulation/features/util/initData.feature@CheckInItem') { itemBarcode: #(extItemBarcode) }
     * def item = checkInResponse.response.item
     And match item.id == itemId
-    And match item.status.name == 'Available'
\ No newline at end of file
+    And match item.status.name == 'Available'
+
+  Scenario: When an item has the status intellectual item, do not allow checkout
+
+    * def extItemBarcode = 'FAT-1007IBC'
+    * def extUserBarcode = 'FAT-1007UBC'
+    * def extStatusName = 'Intellectual item'
+
+    # location and service point setup
+    * call read('classpath:domain/mod-circulation/features/util/initData.feature@PostLocation')
+    * call read('classpath:domain/mod-circulation/features/util/initData.feature@PostServicePoint')
+
+    # post an item
+    * call read('classpath:domain/mod-circulation/features/util/initData.feature@PostInstance')
+    * call read('classpath:domain/mod-circulation/features/util/initData.feature@PostHoldings')
+    * call read('classpath:domain/mod-circulation/features/util/initData.feature@PostItem') { extItemBarcode: #(extItemBarcode), extStatusName: #(extStatusName) }
+
+    # post a user
+    * call read('classpath:domain/mod-circulation/features/util/initData.feature@PostGroup')
+    * call read('classpath:domain/mod-circulation/features/util/initData.feature@PostUser') { extUserBarcode: #(extUserBarcode) }
+
+    # check-out the item and verify that checking-out the item is not processable
+    * def checkOutByBarcodeEntityRequest = read('samples/check-out-by-barcode-entity-request.json')
+    * checkOutByBarcodeEntityRequest.userBarcode = extUserBarcode
+    * checkOutByBarcodeEntityRequest.itemBarcode = extItemBarcode
+    Given path 'circulation', 'check-out-by-barcode'
+    And request checkOutByBarcodeEntityRequest
+    When method POST
+    Then status 422
+    * def error = response.errors[0]
+    And match error.message == '#string'
```

Comment:
```suggestion
    And match error.message contains 'the item status Intellectual item and cannot be checked out'
```

Reply:
Alexander wanted me to check whole message, which includes variable material type and item barcode, or it is enough to check error message like this?

Reply:
ok, let's do it as Alexander asked you

Reply:
@alexanderkurash @viktor-draban , fixed